lib/motors/controller.cpp

```diff
@@ -76,68 +76,61 @@ void Controller::processErrorMessage(const std::uint16_t error_code)
 
 ControllerStatus Controller::processWarningMessage(const std::uint8_t warning_code)
 {
-  // Flag specific warnings in binary
+  ControllerStatus priority_error = ControllerStatus::kNominal;
+
+  // In the event any warning are found, print entire error code.
+  if (warning_code != 0) {
+    logger_.log(core::LogLevel::kInfo, "Controller Error found, (code: %x)", warning_code);
+  } else {
+    return priority_error;
+  }
```

Comment:
better to do if warning == 0 return
then log error code after the if statement finishes